lib/jxl/common.h

```diff
@@ -30,6 +30,15 @@
 namespace jxl {
 // Some enums and typedefs used by more than one header file.
 
+// Define MEMORY_SANITIZER if memory sanitizer is enabled.
```

Comment:
MEMORY_SANITIZER enables a bunch of other behavior which we should re-evaluate, in particular the initialization of padding, which may not be an issue with recent versions. This should probably be in a different commit.

Reply:
I could make a separate commit, but the fix won't work without this change.
IIUC MEMORY_SANITIZER macro is a legacy way, currently documentation says "In some cases one may need to execute different code depending on whether MemorySanitizer is enabled. __has_feature can be used for this purpose.". In brotli we do check both macro and has_feature to detect MSAN build.

Reply:
Or we could move this to compiler options.